examples/nlp/language_modeling/tuning/megatron_gpt_peft_eval.py

```diff
@@ -125,8 +125,20 @@ def main(cfg) -> None:
         # update the model config of the trained model with params we want to set at inference time.
         peft_model_cfg.precision = cfg.trainer.precision
         peft_model_cfg.data.test_ds = cfg.model.data.test_ds
```

Comment:
could you pls add , since it otherwise uses the global batch size of pretrained model instead of configuration

        peft_model_cfg.global_batch_size = cfg.model.global_batch_size
        peft_model_cfg.micro_batch_size = cfg.model.micro_batch_size

Reply:
don't we use `model.data.test_ds.global_batch_size` to set this parameter?